docs/readthedocs/source/doc/Friesian/QuickStart/Train_2tower.md

```diff
@@ -0,0 +1,113 @@
+### Preprocess and Train Two Tower Model Using Movielens Data
+
+Step 1. Initialize OrcaContext
+```python
+sc = init_orca_context("local",  cores=8, memory="8g", init_ray_on_spark=True)
+```
+
+Step 2. Creating a ratings table, user table and an item table
+```python
+from bigdl.friesian.feature import FeatureTable
+from bigdl.dllib.feature.dataset import movielens
+import pandas as pd
+
+data_dir = "./movielens"
+ratings = movielens.get_id_ratings(data_dir)
+ratings = pd.DataFrame(ratings, columns=["user", "item", "rate"])      
+ratings_tbl = FeatureTable.from_pandas(ratings)
```

Comment:
don't use `from_pandas`; us `read_csv` instead

Reply:
Currently, spark 2.4.6 does not support a separator of "::" while reading csv files, it only supports a separator of one character.